plugins/azure/keyvaults/sslCertificateAutoRenewal.js

```diff
@@ -0,0 +1,75 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'SSL Certificate Auto Renewal',
+    category: 'Key Vaults',
+    domain: 'Identity and Access Management',
+    description: 'Ensures that Microsoft Azure Key Vault SSL certificates auto renewal is enabled.',
+    more_info: 'Configure SSL certificates for auto renewal in order to prevent any application or service outage, credential leak, or process violation that can disrupt your business.',
```

Comment:
```suggestion
    more_info: 'Configure auto renewal for SSL certificates in order to prevent any application or service outage, credential leak, or process violation that can disrupt your business.',
```